.github/workflows/check-labels.yml

```
@@ -1,7 +1,7 @@
 name: check-labels
 on:
   pull_request:
-    types: [opened, reopened, labeled, unlabeled]
+    types: [opened, reopened, labeled, unlabeled, synchronize, ]
```

Either

```suggestion
    types: [opened, reopened, labeled, unlabeled, synchronize]
```

or

```suggestion
    types:
    - opened
    - reopened
    - labeled
    - unlabeled
    - synchronize
```

?

---

(either have a clean, short list or keep it extensible as a bulleted list)